src/WheelsAndGoods.Application/Models/Paginations/PaginationOrderRequest.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace WheelsAndGoods.Application.Models.Paginations;
+
+public class PaginationOrderRequest : PaginationBase
```

Comment:
Looks like we actually can use just PaginationRequest (rename the PaginationBase) everywhere.